.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: required
 language: java
 jdk:
   - oraclejdk8
+  - oraclejdk9
```

Comment:
Is JDK9 the last one available on Travis?

Reply:
yes, JDK 10 still unavailable on Travis

https://github.com/travis-ci/travis-ci/issues/9368